Codigo/GameLogic.bas

```diff
@@ -2135,8 +2135,77 @@ Public Sub TimerQuestOrco()
     Call SendData(SendTarget.ToAll, 0, PrepareMessagePlayWave(156, NO_3D_SOUND, NO_3D_SOUND))
 End Sub
 
+Public Function TestRequiredEquipedItem(ByRef inventory As t_Inventario, ByVal requiredItemsFlag As Long) As e_EquipedSlotMask
```

Comment:
The name `TestRequiredEquipedItem` is not good.

I'd suggest something more descriptive as `GetEquipedSlotMaskFromUserInv`

Reply:
that doesn't reflect what the function does and it might be more confusing

maybe ValidateRequiredSlotWithUserInv?